### ISSUE TITLE
TF Nightly 20190418 Crashes on Estimator Import

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/tensorflow#27955]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): **No**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): **Ubuntu 16.04**
- TensorFlow installed from (source or binary): **Binary**
- TensorFlow version (use command below): **tf-nightly-20190418**
- Python version: **3.6**

**Describe the current behavior**
Nightly tensorflow crashes on importing estimator-nightly

**Describe the expected behavior**
Able to import

**Code to reproduce the issue**
```
pip install tf-nightly==1.14.1.dev20190418
import tensorflow
```
**Other info / logs**
https://colab.research.google.com/drive/1mvVTcAWHwJY1Kko49Ae93iQum6lT9W8D

Crash:
```
TypeError                                 Traceback (most recent call last)
<ipython-input-2-64156d691fe5> in <module>()
----> 1 import tensorflow as tf

/usr/local/lib/python3.6/dist-packages/tensorflow/__init__.py in <module>()
     32 from tensorflow._api.v1 import autograph
     33 from tensorflow._api.v1 import bitwise
---> 34 from tensorflow._api.v1 import compat
     35 from tensorflow._api.v1 import config
     36 from tensorflow._api.v1 import data

/usr/local/lib/python3.6/dist-packages/tensorflow/_api/v1/compat/__init__.py in <module>()
     19 from __future__ import print_function as _print_function
     20 
---> 21 from tensorflow._api.v1.compat import v1
     22 from tensorflow._api.v1.compat import v2
     23 from tensorflow.python.compat.compat import forward_compatibility_horizon

/usr/local/lib/python3.6/dist-packages/tensorflow/_api/v1/compat/v1/__init__.py in <module>()
    641     parent_package_str=__name__,
    642     child_package_str=(
--> 643         'tensorflow_estimator.python.estimator.api._v1.estimator'))
    644 _component_api_helper.package_hook(
    645     parent_package_str=__name__,

/usr/local/lib/python3.6/dist-packages/tensorflow/python/tools/component_api_helper.py in package_hook(parent_package_str, child_package_str, error_msg)
     54   parent_pkg = importlib.import_module(parent_package_str)
     55   try:
---> 56     child_pkg = importlib.import_module(child_package_str)
     57   except ImportError:
     58     if error_msg:

/usr/lib/python3.6/importlib/__init__.py in import_module(name, package)
    124                 break
    125             level += 1
--> 126     return _bootstrap._gcd_import(name[level:], package, level)
    127 
    128 

/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/__init__.py in <module>()
      6 from __future__ import print_function as _print_function
      7 
----> 8 from tensorflow_estimator._api.v1 import estimator
      9 _names_with_underscore = []
     10 __all__ = [_s for _s in dir() if not _s.startswith('_')]

/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/_api/v1/estimator/__init__.py in <module>()
      6 from __future__ import print_function as _print_function
      7 
----> 8 from tensorflow_estimator._api.v1.estimator import experimental
      9 from tensorflow_estimator._api.v1.estimator import export
     10 from tensorflow_estimator._api.v1.estimator import inputs

/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/_api/v1/estimator/experimental/__init__.py in <module>()
     33 if not isinstance(_sys.modules[__name__], _deprecation_wrapper.DeprecationWrapper):
     34   _sys.modules[__name__] = _deprecation_wrapper.DeprecationWrapper(
---> 35       _sys.modules[__name__], "estimator.experimental", _DEPRECATED_TO_CANONICAL)

TypeError: __init__() takes 3 positional arguments but 4 were given
```

**Comment by krutartha** [comment]
I have the same error while using tensorflowjs converter!

**Comment by villeh1** [comment]
I saw the same issue. Wonder why build process did not catch this issue?

**Comment by kaeflint** [comment]
I had thesame problem. For now installing an older version such as: 
pip install -q tf-nightly-gpu-2.0-preview==2.0.0.dev20190416
solved allowed me to run my models.

**Comment by drewszurko** [comment]
I have the same issue on macOS.

**System information**

- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): **No**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): **macOS 10.14.4**
- TensorFlow installed from (source or binary): **binary**
- TensorFlow version (use command below): **tf-nightly-20190418**
- Python version: **3.6.8**

**Comment by mihaimaruseac** [comment]
It's been already fixed by https://github.com/tensorflow/tensorflow/commit/e84d155152ab5a187d2c0e4c5eea6a73df3789e1 but the nightly process for estimator ran with the code before this commit and the nightly process for tensorflow ran with the code after.

**Comment by mihaimaruseac** [comment]
For now, if you want a quick fix, you can edit the `tensorflow/python/util/deprecation_wrapper.py` file to have:

```python
class DeprecationWrapper(types.ModuleType):
  """Wrapper for TensorFlow modules to support deprecation messages."""

  def __init__(self, wrapped, module_name, _unused_arg={}):  # pylint: disable=super-on-old-class
```

(just add the `_unused_arg={}` argument). Though this is just patching, and will have to be undone on the next nightly.

**Comment by qlzh727** [comment]
Can we rerun the nightly process for estimator so that user don't have to patch and revert? Also since most of library rely on tf from PIP install, which is hard to patch every time.

**Comment by mihaimaruseac** [comment]
Unfortunately if we launch the nightly process now we will have issues with automatic tooling on the next nightly, so probably it's better to either patch or use an older nightly.

**Comment by seanpmorgan** [comment]
Fixed in 20190419 nightly. Thanks!
